### PR TITLE
Add Laravel 6 support

### PR DESCRIPTION
**Pull request by tarasvi**
- Move assets

- Add logging.php

- Add hashing.php

- Cleanup

- Update .env file

- Rename AWS env variables

- Update composer

- Adopt Laravel coding style

### CODE REVIEW

File: resources/lang/en/passwords.php [hunk 1]
```diff
@@ -13,7 +13,6 @@
     |
     */
 
-    'password' => 'Passwords must be at least six characters and match the confirmation.',
```
[the remark this answers is not in the document]
Comment (tarasvi) [reply resources/lang/en/passwords.php]: It was removed by the Laravel Shift service. I didn't see any password issue related.

File: bootstrap/app.php [hunk 2]
```diff
@@ -48,30 +43,6 @@
...
-        if ($mode === 'single') {
-            $handler = new StreamHandler($logFile);
-        } else if ($mode === 'errorlog') {
-            $handler = new ErrorLogHandler();
-        } else {
-            $handler = new RotatingFileHandler($logFile, 5);
-        }
-
-        $monolog->pushHandler($handler);
-        $handler->setFormatter(new LineFormatter(null, null, true, true));
```
[the remark this answers is not in the document]
Comment (tarasvi) [reply bootstrap/app.php]: Now logging functionality is placed in [config/logging.php](https://github.com/dreamfactorysoftware/dreamfactory/pull/305/files#diff-0c5eb58e57861e3bdcc3a2f6ae493626) file. This [logging.php](https://github.com/dreamfactorysoftware/dreamfactory/pull/305/files#diff-0c5eb58e57861e3bdcc3a2f6ae493626) was not present in Laravel 5.5.

File: .env-dist [hunk 3]
```diff
@@ -20,13 +31,13 @@ APP_ENV=local
 ## Use 'php artisan key:generate' to generate a new key. Key size must be 16, 24 or 32.
 APP_KEY=
 #APP_LOCALE=en
-## LOG setting. Where and/or how the log file is setup. Options are single (default), daily, syslog, errorlog
-#APP_LOG=single
+## Here you may configure the log channels for your application. Options are stack (default), slack, single, daily, syslog, errorlog
+#LOG_CHANNEL=stack
```
[the remark this answers is not in the document]
Comment (tarasvi) [reply .env-dist]: Yes, slack and single are different.
**Single** is just a type of logging. **Stack** is wrapper under different types of logging.
You can find more details here:
https://laravel.com/docs/5.8/logging#configuration
https://img.in6k.com/screens/eefff04f_2019.24.10.png
**Slack** is also just one of the types of logging.

File: .env-dist [hunk 4]
```diff
@@ -20,13 +31,13 @@ APP_ENV=local
...
 ## LOG Level. This is hierarchical and goes in the following order.
 ## DEBUG -> INFO -> NOTICE -> WARNING -> ERROR -> CRITICAL -> ALERT -> EMERGENCY
 ## If you set log level to WARNING then all WARNING, ERROR, CRITICAL, ALERT, and EMERGENCY
 ## will be logged. Setting log level to DEBUG will log everything.
-#APP_LOG_LEVEL=warning
+APP_LOG_LEVEL=warning
```
Comment (yaroslavmo) [inline .env-dist]: Shouldn't we leave it commented? so then in the config, it takes default value.

Comment (tarasvi) [reply .env-dist]: @yaroslavmo, yes. you are right. I fixed this.

**Pull request merged**